### ISSUE TITLE
weapon and potion balancing

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#213]
```
Dreia and others stated many usefull things here:
http://andors.techby2guys.com/viewtopic.php?f=4&t=333
http://andors.techby2guys.com/viewtopic.php?f=4&t=854

I think we really should look into this.

As qasur stated:
In your starting village, which looks like a Farming village, you've got the 
majority of the weapons available in the game right there.

We should improve items that are currently useless like big hammers and we 
should redistribute the items to the different stores.
```

Original issue reported on code.google.com by `SamuelPl...@gmail.com` on 9 Jul 2011 at 7:12

**Comment by GoogleCodeExporter** [comment]
```
Started for v0.6.10.
```

Original comment by `oskar.wi...@gmail.com` on 30 Aug 2011 at 9:04
- Changed state: **Started**
- Added labels: **Type-Enhancement**, **Milestone-0.6.10**, **Priority-Medium**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Ready for v0.6.10.
```

Original comment by `oskar.wi...@gmail.com` on 7 Oct 2011 at 7:54
- Changed state: **Ready**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.10 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 21 Oct 2011 at 8:56
- Changed state: **Fixed**
